server/src/db.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import os
+
+from motor import motor_asyncio
+
+
+DB_CONNECTION = None
+DATABASE = None
+log = logging.getLogger(__name__)
+
+
+def db_connection():
+    global DB_CONNECTION
+    if DB_CONNECTION is None:
+        mongo_url = os.environ['MONGO_URL']
+        DB_CONNECTION = motor_asyncio.AsyncIOMotorClient(mongo_url)
+    return DB_CONNECTION
+
+
+def db() -> motor_asyncio.AsyncIOMotorDatabase:
+    global DATABASE
+    if DATABASE is None:
+        mongo_database = os.environ['MONGO_DATABASE']
+        DATABASE = db_connection()[mongo_database]
```

Comment:
```suggestion
        DATABASE = db_connection()[os.environ['MONGO_DATABASE']]
```

Reply:
Don't really see purpose for one more variable, it's pretty clear, that env variable contains MONGO_DATABASE name. What do you think?